.github/workflows/upgrade_test.yaml

```diff
@@ -0,0 +1,71 @@
+name: Upgrade functionality test
+
+on: 
+  push:
+    branches: [ upgrade-testing ]
+
+jobs:
+  
+  upgrade-functionality-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout action sources
+      uses: actions/checkout@v2.3.4
+
+    - name: Checkout fission sources
```

Comment:
We should not clone examples repo, as we need to single file from repo only later and it's fetched via curl later.

Reply:
Removed examples repo from workflow.